scvi/module/_jaxvae.py

```diff
@@ -125,16 +117,20 @@ def setup(self):
             n_latent=self.n_latent,
             n_hidden=self.n_hidden,
             dropout_rate=self.dropout_rate,
-            is_training=self.is_training,
         )
 
         self.decoder = FlaxDecoder(
             n_input=self.n_input,
             dropout_rate=0.0,
             n_hidden=self.n_hidden,
-            is_training=self.is_training,
         )
 
+    def train(self):
```

Comment:
I don't think this is ever called right now? This PR should require changes to JaxModuleWrapper which has train() eval() functions. The trainer calls train() and eval() on the JaxModuleWrapper which currently wont be doing anything with respect to this parameter. Definitely add some tests to ensure this is working before going further.